test/user.test.js

```diff
@@ -75,20 +76,23 @@ describe('user routes', () =>{
             const res2 = await request.get('/api/users');
             expect(res2.body[3].preferredPaymentMethod).toBe('CASH')
         })
+
         // //ATM UUID is passed in with making a post request, but should change this so a UUID is generated for the user
-        it('should throw an error if uuid, username, password is omitted', () =>{
-                try{  
-                    const res = request
-                    .post('/api/users')
-                    .send({
-                        uuid: '6',
-                        password: 'testPassF'
-                    })
-                }
-                catch(err){
-                    expect(err.message).toBe('userName required')
-                }
-        })
+        // it('should throw an error if uuid, username, password is omitted', () =>{
```

Comment:
I know you explained it in the description, but should there be a TODO comment here as well to explain the comments/issues?

Reply:
Yeah that's a good point but in this case, there's already a ticket in the backlog for this and it should be picked up immediately after. Additionally, I'm already sort of investigating it. I don't think TODO is necessary.

We can reinforce TODO comments from herein though.

Reply:
Okay sounds good  👍